demo/transient/transient_snes_mpfaof90.F90

```diff
@@ -305,7 +312,8 @@ program main
   CHKERRA(ierr);
 
   if (pflotran_consistent) then
-     call TDySetPorosityFunction(tdy,PorosityFunctionPFLOTRAN,0,ierr);
+!     call TDySetPorosityFunction(tdy,PorosityFunctionPFLOTRAN,0,ierr);
```

Comment:
Will be eventually make `TDySetPorosityFunction` an internal TDy lib function and not expose it to the world?

Reply:
Sure. If people like the registry idea and want to use it exclusively, I see no reason we couldn't make this and the other similar functions internal/private.